packages/library-base/src/commonMain/kotlin/io/realm/internal/RealmImpl.kt

```diff
@@ -118,6 +118,20 @@ public class RealmImpl private constructor(
             }
         )
 
+    /**
+     * Manually force this Realm to update to the latest version.
+     * The refresh will also trigger any relevant notifications.
+     * TODO Public because it is called from `SyncSessionImpl`.
+     */
+    public suspend fun refresh() {
```

Comment:
This concept is a leftover from the old architecture and I feel is alien to the new unidirectional frozen architecture, we should strive to keep this method internal/out of the public API